### ISSUE TITLE
Long click edit links (to modal) in sidenav-tree need to break out of iframe

### ISSUE THREAD

**Issue by adrianbj** [ryancramerdesign/AdminThemeUikit#11]
As per title - currently they open in the page tree sidebar - just very useful :)

**Comment by adrianbj** [comment]
More follow up - even View links in the page list in main content iframe don't work as expected - they open in the iframe. And then there is also the View "tab" on the edit page - the default option now opens in the iframe and of course the dropdown alternatives are also in need of some adjustments.

**Comment by ryancramerdesign** [comment]
My plan is to drop the long-click support for this sidebar usage, as these sidebar layouts effectively kill the need for it. Though maybe longclick for view will remain to open in main. As for the view links opening in the main frame, I thought that was a benefit, but can see what you mean, as it's not consistent with what people are used to here. I've updated it so that the view links work like before, opening in _top rather than in main.

**Comment by adrianbj** [comment]
I think that viewing the page in the main frame has some advantages for sure, but it could result in someone loses unsaved edits and so it needs to be considered carefully, or made more obvious, or something :)

**Comment by ryancramerdesign** [comment]
Btw, since you are here, just wanted to mention that the new layouts you asked for last week are now there in the latest version. I agree with you, the masthead navigation + sidebar page tree seems like the best combination if using a sidebar.

**Comment by adrianbj** [comment]
Yeah, been playing with it a bit already :) It seems like a great combo if we can get link integrity issues sorted out.

**Comment by adrianbj** [comment]
@ryancramerdesign - almost all issues in v6 seem to be fixed - I like how the long click on view works with a fullscreen modal now.

The one remaining scenario that I think is still a little weird is if you load the admin and then open the sidebar. Now you have the page tree in the sidebar, as well as the main content panel - like this:

![screen shot 2017-04-01 at 7 56 58 am](https://cloud.githubusercontent.com/assets/871211/24579794/d863ab7c-16b0-11e7-851a-12f54a2157aa.png)



Now the issue of long clicks on the page list actions appears again. 

The other possible improvement here might be to actually have an option for the view link in the sidebar to view the page in the main frame - I think this could actually be quite a useful default, so long as it's obvious how to easily view it full page as well.

**Comment by ryancramerdesign** [comment]
> The one remaining scenario that I think is still a little weird is if you load the admin and then open the sidebar. 

I've set it up so that it automatically closes the sidebar if it's open, when you go to either PageList or PageLister. Meaning, you'd have to manually open it up if you are in either of those processes. Now I could hide it completely with no ability to open it up, but maybe there's a use case for being able to see two trees side-by-side. So currently leaning towards leaving it there to the user (who would have to manually open the sidebar if they wanted it).

**Comment by adrianbj** [comment]
I think this leads back to my idea of having the ability to replace the page tree with something else (maybe a Lister Pro view or some other Process). In that case I think have the sidebar open along with the main panel would make sense, but certainly not when both are the PageList
